### ISSUE TITLE
Implementation: Part 3 pretrained PyTorch classification model

### ISSUE THREAD

**Issue by prajnan93** [prajnan93/filter-predictor#7]
1. food
2. pets
3. portraits(labels humas)
4. landscape (sky, mountain, city, building)

`pytorch`
